tests/test_install.py

```diff
@@ -361,3 +361,10 @@ def test_install_run_in_separate_directory(caplog, capsys, pipx_temp_env, monkey
     f.touch()
 
     install_packages(capsys, pipx_temp_env, caplog, ["pycowsay"], ["pycowsay"])
+
+
+def test_force_install_python_flag_warning(pipx_temp_env, capsys):
+    assert not run_pipx_cli(["install", "pycowsay"])
+    assert not run_pipx_cli(["install", "pycowsay", "--force"])
+    captured = capsys.readouterr()
+    assert "--python is ignored when --force is passed." not in captured.out
```

Comment:
I don't think this needs it's own test case. Just changing the existing test like so should do the trick IMO. 

```python
def test_passed_python_and_force_flag_warning(pipx_temp_env, capsys):
    assert not run_pipx_cli(["install", "black"])
    assert not run_pipx_cli(["install", "--python", sys.executable, "--force", "black"])
    captured = capsys.readouterr()
    assert "--python is ignored when --force is passed." in captured.out

    assert not run_pipx_cli(["install", "black", "--force"])
    captured = capsys.readouterr()
    assert "--python is ignored when --force is passed." not in captured.out, "Should only print warning if both flags present"

    assert not run_pipx_cli(["install", "pycowsay", "--force"])
    captured = capsys.readouterr()
    assert "--python is ignored when --force is passed." not in captured.out, "Should not print warning if package does not exist yet"
```

Reply:
Modified the test case and removed `test_force_install_python_flag_warning`